apps/btshell/src/cmd.c

```diff
@@ -596,7 +597,6 @@ static const struct shell_param connect_params[] = {
     {"timeout", "usage: =[UINT16], default: 0x0100"},
     {"min_conn_event_len", "usage: =[UINT16], default: 0x0010"},
     {"max_conn_event_len", "usage: =[UINT16], default: 0x0300"},
-    {"duration", "usage: =[1-INT32_MAX], default: 0"},
```

Comment:
this change looks good, but it would be nice to have comment why it is needed as it does not change functionality.